.github/workflows/ci.yml

```
@@ -59,10 +59,15 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+          submodules: recursive
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
           go-version-file: "go.mod"
+      - name: Set up solc
+        uses: ARR4N/setup-solc@v0.2.0
+        with:
+          version: '0.8.30'
```

```suggestion
          versions: '0.8.30'
```

The singular will install it as `solc`, which doesn't lock in a version at the point of use. The plural will force use of a specific version as `solc-v0.8.30`, which is important when checking in generated code.